repos/aspnet-extensions.proj

```diff
@@ -52,7 +52,7 @@
     <UseSourceBuiltSdkOverride Include="@(ArcadeSdkOverride)" />
   </ItemGroup>
 
-  <Target Name="SetOutputList" AfterTargets="Build" BeforeTargets="GatherBuiltPackages">
+  <Target Name="SetOutputList" AfterTargets="Package" BeforeTargets="GatherBuiltPackages">
```

Comment:
Was this incorrect when aspnet extensions was added or is this being changed to fix something with aspnet core tooling?